src/main/java/qna/domain/Question.java

```diff
@@ -88,6 +89,22 @@ public boolean isDeleted() {
         return answers;
     }
 
+    public DeleteHistories delete(User user) throws CannotDeleteException {
+        DeleteHistories deleteHistories = new DeleteHistories();
+        deleteValidation(user);
+        this.deleted = true;
+        deleteHistories.addDeleteHistory(new DeleteHistory(ContentType.QUESTION, getId(), getWriter()));
+        Answers.of(getAnswers()).deleteALL(user, deleteHistories);
```

Comment:
```suggestion
        deleteValidation(user);
        
        this.deleted = true;
        
        DeleteHistories deleteHistories = new DeleteHistories();
        deleteHistories.addDeleteHistory(new DeleteHistory(ContentType.QUESTION, getId(), getWriter()));
        
        Answers.of(getAnswers()).deleteALL(user, deleteHistories);
```
와 같이 공백을 사용해서 문단을 나눠 보세요. 
그리고 지역변수는 사용되는 시점에 생성하는것을 권장 드립니다. 
https://ktko.tistory.com/entry/Effective-Java-45-%EC%A7%80%EC%97%AD-%EB%B3%80%EC%88%98%EC%9D%98-%EC%9C%A0%ED%9A%A8%EB%B2%94%EC%9C%84%EB%A5%BC-%EC%B5%9C%EC%86%8C%ED%99%94%ED%95%98%EB%9D%BC